xds/internal/httpfilter/fault/fault.go

```diff
@@ -125,7 +125,12 @@ func (builder) BuildClientInterceptor(cfg, override httpfilter.FilterConfig) (ir
 		}
 	}
 
-	return &interceptor{config: c.config}, nil
+	icfg := c.config
+	if (icfg.GetMaxActiveFaults() != nil && icfg.GetMaxActiveFaults().GetValue() == 0) ||
```

Comment:
`GetValue()` does handle a `nil` receiver.
```golang
func (x *UInt32Value) GetValue() uint32 {
	if x != nil {
		return x.Value
	}
	return 0
}
```
Do, we still need the `icfg.GetMaxActiveFaults() != nil` check?

Reply:
Yes.  If `GetMaxActiveFaults` is _explicitly_ set to zero, it is disabled.  But if it is unset, there is no limit.